README.md

```diff
@@ -131,6 +133,9 @@ services:
       - LLDAP_JWT_SECRET=REPLACE_WITH_RANDOM
       - LLDAP_LDAP_USER_PASS=REPLACE_WITH_PASSWORD
       - LLDAP_LDAP_BASE_DN=dc=example,dc=com
+      # If using LDAPS, configure your cert and keys
+      # - LLDAP_LDAPS_OPTIONS__CERT_FILE=PATH_TO_CERT
```

Comment:
Maybe we can add the file extension? I sometimes get confused between the cert, pem and key file (well, mostly about the pem file, whether it's a cert of key)

Reply:
Im taking over this, and will re-request review